common/writeThoughFile_windows.go

```diff
@@ -38,7 +38,7 @@ func GetFileInformation(path string) (windows.ByHandleFileInformation, error) {
 	}
 	// custom open call, because must specify FILE_FLAG_BACKUP_SEMANTICS when getting information of folders (else GetFileInformationByHandle will fail)
 	fd, err := windows.CreateFile(srcPtr,
-		windows.GENERIC_READ, windows.FILE_SHARE_READ, nil,
+		windows.GENERIC_READ, windows.FILE_SHARE_READ|windows.FILE_SHARE_WRITE|windows.FILE_SHARE_DELETE, nil,
```

Comment:
For the first reason above, this is necessary for the download scenario, because, within the same process, multiple times after opening here, we may potentially open the file for writing again (write attributes is where my brain mostly goes)

Reply:
Nope, actually, it wasn't, I just tried it after thinking I should.